cloudformation/batch-substack.cfn.json

```diff
@@ -1048,6 +1056,12 @@
         "Ref": "ResourcesS3Bucket"
       }
     },
+    "ArtifactS3RootDirectory": {
```

Comment:
I don't think we need it as an output here

Reply:
Since we are outputting `ResourcesS3Bucket` I thought it would be nice to keep things consistent and output `ArtifactS3RootDirectory ` as well, so people always know where batch resources are stored when looking only at the batch substack.
I can remove if this is not necessary.